README.md

```diff
@@ -232,7 +232,7 @@ if err != nil {
 }
 
 // You can also publish a batch of messages in a single request.
-err = channel.PublishBatch(ctx, []*ably.Message{
+err = channel.PublishMultiple(ctx, []*ably.Message{
```

Comment:
Perhaps change the comment to "publish multiple messages" so not to confuse with a batch publish?

Reply:
Done at 30a6413.